packages/hdwallet-ledger-webusb/src/transport.ts

```diff
@@ -151,6 +151,8 @@ export class LedgerWebUsbTransport extends LedgerTransport {
     return core.mustBeDefined(this.device.serialNumber);
   }
 
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore of course it's @ledgerhq types mismatches again
```

Comment:
Heh, "don't lint the next line of TS please. Oh, and let me comment on the next request I make for for the language server to ignore the next line of TS."